### DISCUSSION TITLE
Create actual folders on device for albums

### DISCUSSION THREAD

**Discussion by EcksDy** [ente-io/ente#4336] [Enhancements]
I've noticed an unintuitive behaviour, where albums created in the ente app will not be created in the file system. 
This also affects items moved to albums through ente app, moving items between albums won't actually move them in the file system.

Look at the following scenario:

I've edited a clip that was synced in an album, i.e.: Camera. When saving, it creates a new file. The file will go to a newly created folder/album named Movies.
Movies and the edited clip are not synced, as it's a new folder.

I move the edited clip to Camera(synced folder), it's where the original clip is. Now the edited clip shows up in both places, once in Camera and once in Movies. The one that is in Camera is getting synced, while the one in Movies has the no cloud icon on it.

It lead me to believe they are 2 separate files, in 2 separate folders. When the sync finished for the file in Camera, I've deleted the clip in Movies. I did not expect it to delete the file in Camera too.

The detachment from the file system makes me rely on the default samsung galaxy gallery app. As of now, I'm using it to create albums(which will create folders), and when moving files to albums(folders) it will move the actual files too. Then I go to ente and add the folder to backup. 

This will lead to problems down the road, as I'm syncing the default album Camera with ente. Using the other gallery app to move files into their own albums/folders, and then syncing them will create dups in the new album and the default Camera one.

I would love ente to fulfill it's intended purpose of being the replacement gallery app and backup service, without the need for external apps.

**Comment by iadaz** [comment]
I have a similar suggestion to reduce the number of upload 'errors': https://github.com/ente-io/ente/discussions/2878
